packages/toggle-core/src/Read/AvailableTypes.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pheature\Core\Toggle\Read;
+
+interface AvailableTypes
```

Comment:
What do you think about renaming this interface to something that describes the capability of the implementation to process or build fixed types? I'm not sure if this name fits good or maybe we should use another one.. 🤔

```suggestion
interface WithProcessableFixedTypes
```

Reply:
I understand that we need this method in a different interface because the user/dev could not want to define it but if it's implemented, we'll merge those types with the ones we have in the core...

Reply:
That's right ;-D, also, you can create a custom factory to use out of the chain factory, and in that case, the array of available strategy types should not be needed.